Tests/CTestList.cmake

```diff
@@ -38,6 +38,7 @@ function(add_test_r TEST_NAME TEST_EXE PLTFILE)
     set(TEST_EXE ${CMAKE_BINARY_DIR}/Exec/${TEST_EXE})
     set(FCOMPARE_TOLERANCE "-r 1e-10 --abs_tol 1.0e-12")
     set(FCOMPARE_FLAGS "-a ${FCOMPARE_TOLERANCE}")
+    # TODO: Check if we need MPI_COMMANDS for running FCOMPARE_EXE
```

Comment:
Agree, I don't think this is necessary. @rafmudaf is it better to just leave it in?

Reply:
Good point, lets remove that just for our own future clarity.

Reply:
Oh it looks like this was merged before this comment was addressed. I'll create a new pull request for this.